### PR TITLE
EDX-1383: Details screen UI fixes

### PR DESCRIPTION
**Pull request by SodhiA1**

### CODE REVIEW

File: frontend/src/components/institute/common/SchoolMove.vue [hunk 1]
```diff
@@ -1,22 +1,39 @@
 <template>
   <div id="schoolMove" class="px-0 pt-3 ma-0" style="width: 100%;">
-    <v-row>
-      <v-col class="d-flex justify-end">
-      <PrimaryButton v-if="isMoveSchoolAllowed()" id="moveSchoolButton" class="mr-2" secondary icon-left
-                               icon="mdi-arrow-left-right" @click.native="moveSchool"
-                               text="Move School"></PrimaryButton>
+    <v-row v-if="loading">
```
Comment (trev-dev) [inline frontend/src/components/institute/common/SchoolMove.vue:3]: The HTML markup tree seems out of alignment to me, thus making it harder to follow.  It took a minute for me to see which closing row tags matched the opening row tags to ensure everything was opened & closed.

Could we give this file a bit of quick re-format?

**Review by trev-dev — CHANGES_REQUESTED** [review]

File: frontend/src/components/institute/common/SchoolMove.vue (hunk 1 above)
Comment (trev-dev) [reply frontend/src/components/institute/common/SchoolMove.vue:3]: We got this.

File: frontend/src/components/institute/SchoolDetails.vue [hunk 2]
```diff
@@ -270,4 +267,13 @@ export default {
     font-weight: bold;
   }
 
+  .tab-divider {
+    border-right: 1px solid lightgray;
```
Comment (trev-dev) [inline frontend/src/components/institute/SchoolDetails.vue:271]: Me being smart.

File: frontend/src/components/institute/SchoolDetails.vue [hunk 3]
```diff
@@ -48,7 +48,7 @@
         </v-row>
         <v-row>
             <v-tabs active-class="active-display" v-model="tab">
-              <v-tab v-for="item in items" :key="item">
+              <v-tab class="tab-divider" v-for="item in items" :key="item">
```
Comment (arcshiftsolutions) [inline frontend/src/components/institute/SchoolDetails.vue:51]: This is a test

**Review by trev-dev — APPROVED** [review]

**Pull request merged**